Dalamud/Game/Internal/Framework.cs

```diff
@@ -101,6 +104,7 @@ public sealed class Framework : IDisposable {
 
         private bool HandleFrameworkUpdate(IntPtr framework) {
             try {
+                dalamud.ClientState.Actors.ResetCache();
```

Comment:
I think we should subscribe to this from ActorTable, instead of doing it in the hook itself

Reply:
The reason I did it there is because its done all the way before the event handler is invoked. That way I can guarantee the cache is reset before any calls to the actor table functions happen. I'll see if it works fine from within and commit the change if it is.

Reply:
Surprisingly it didn't cause any side effects. Worse case would be that the actor table would be a frame behind, but I didn't see it manifesting and it wouldn't be a major issue.